### DISCUSSION TITLE
استفاده از DoH برای دور زدن فیلترینگ

### DISCUSSION THREAD

**Discussion by TKZ700** [GFW-knocker/gfw_resist_tls_proxy#98] [Ideas]
بسته شد

**Comment by GFW-knocker** [comment]
چون ایپی های یوتیوب  و توییتر فیلتر نیست و فقط دامنه اش فیلتره میشه دور زد اونم رو ایرانسل
موقت جواب میده ولی روش پایدار و اصولی نیست
اینکه  برخی ایپی های یوتیوب بازه چون میخوان گوگل باز باشه نبستن وگرنه میبستن
